unicodetools/data/ucd/dev/DerivedCoreProperties.txt

```diff
@@ -12954,6 +13035,8 @@ FE20..FE2F    ; InCB; Extend # Mn  [16] COMBINING LIGATURE LEFT HALF..COMBINING
 1133B..1133C  ; InCB; Extend # Mn   [2] COMBINING BINDU BELOW..GRANTHA SIGN NUKTA
 11366..1136C  ; InCB; Extend # Mn   [7] COMBINING GRANTHA DIGIT ZERO..COMBINING GRANTHA DIGIT SIX
 11370..11374  ; InCB; Extend # Mn   [5] COMBINING GRANTHA LETTER A..COMBINING GRANTHA LETTER PA
+113CE         ; InCB; Extend # Mn       TULU-TIGALARI SIGN VIRAMA
```

Comment:
Why is 113CF not here?

Reply:
Because the looped virama is not gcb=Extend, because it's not Grapheme_Extend, because it's gc=Mc and not Other_Grapheme_Extend.